contribute/tool_resource_update.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Updating the tool and resource list
+summary: How to add a tool or resource to Infectious Diseases Toolkit
+---
+
+## Way of working
+
+The tools or resources you will find on pages are a filtered set from a [bigger list](all_tools_and_resources). This filtering is done using page_id. If a tool or resource is tagged with for example the page_id `metadata`, it will be automatically listed on the corresponding page. The page_id can be found in the [Website overview page](/contribute/website_overview). 
```

Comment:
```suggestion
The tools or resources you will find on pages are a filtered set from a [bigger list](all_tools_and_resources). This filtering is done using page_id. If a tool or resource is tagged with, for example, the page_id `metadata`, it will be automatically listed on the corresponding page. The page_id can be found in the [Website overview page](/contribute/website_overview). 
```